salt/modules/file.py

```diff
@@ -6243,6 +6243,7 @@ def list_backups(path, limit=None):
         [files[x] for x in sorted(files, reverse=True)[:limit]]
     )))
 
+
```

Comment:
Can we pull newline out please? I'm not sure why it was added.

Reply:
Sure, but this is needed. Comes from a flake8 / pep8 complaint (E305: E305 expected 2 blank lines after class or function definition). After each function we need to add two lines. Looks like that there is no exception for assigments.